scripts/validate_docstrings.py

```diff
@@ -75,7 +97,7 @@ class Docstring:
     def __init__(self, method_name, method_obj):
         self.method_name = method_name
         self.method_obj = method_obj
-        self.raw_doc = pydoc.getdoc(method_obj)
+        self.raw_doc = method_obj.__doc__ or ''
```

Comment:
Is there a reason you changed this back? 
An advantage of `pydoc.getdoc` is that it correctly deals with dedenting docstrings

Reply:
One of the reasons I made this change is because when the dedenting is not done properly, the inference of the parameters by NumpyDocString goes wrong (eg try to validate pandas.Timestamp)

Reply:
`pydoc.getdoc` strips the string, so we can't validate whether the user is starting the docstring after the quotes, in the next line, or 3 lines after. I thought it was only used to dedent the docstring in the print, and I added the `textwrap.dedent` there. Will take a look at what you say in `pandas.Timestamp`.

Reply:
Ah, yes. We can also keep both ... but that makes it a bit more complex.
Or only do the dedented one (with pydoc.getdoc) when passing it to NumpyDocString